types/simpl-schema/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "object-literal-shorthand": false
```

Comment:
Why is this needed?

Reply:
Because of this discussion: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/31054#discussion_r239460272

If have to use `custom: function() { this.value; }` to make the context available:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d01679afa6d11acd93f6adaa914d2ab74cf52afd/types/simpl-schema/simpl-schema-tests.ts#L42